be/src/vec/exec/jni_connector.h

```diff
@@ -197,6 +216,17 @@ class JniConnector {
      */
     Status get_nex_block(Block* block, size_t* read_rows, bool* eof);
 
+    /**
+     * Call java side function JniScanner.getTableSchema.
+     *
+     * The schema information are stored as a string.
+     * Use # between column names and column types.
+     *
+     * like: col_name1,col_name2,col_name3#col_type1,col_type2.col_type3
```

Comment:
There is an issue with the concatenation method of this string, as type may contain commas, such as the decimal type. You can directly return Map<String, String> type, as shown in the process of passing map types as parameters.

Reply:
In the support of complex types, I have rebuild this piece. Like array, map types are nested.